### PR TITLE
add event dispatchers when lists curation & creation happen

### PR DESCRIPTION
**Pull request by iisa**
Dev notes:
- i did not use analytics package as petabox has different api. (send_event instead of sendEvent)
- also, lots of stuff happening in component, hard to parse, so opted for keeping analytics responsibility in petabox.

Testing: Open Demo, open dev console, watch console logs fire

### CODE REVIEW

**Review by latonv — APPROVED** [review]

**Comment by latonv** [comment]
Thanks for keeping responsibilities separated!

File: src/ia-item-user-lists.ts [hunk 1]
```diff
@@ -89,22 +89,50 @@ export class IaItemUserLists extends LitElement {
...
+    const listsSettingsErrorEventListener = (e: CustomEvent) => {
+      // eslint-disable-next-line no-console
+      console.log('userListError listener', e.detail);
+      this.dispatchEvent(
+        new CustomEvent('userListSettingsError', {
+          detail: { ...e.detail },
+        })
+      );
+    };
+    window.addEventListener(
```
Comment (mc2) [inline src/ia-item-user-lists.ts:103]: Question: shouldn't this event listener be removed when the component is disconnected?

**Comment by iisa** [comment]
closing for #15
